### ISSUE TITLE
edge.txt 

### ISSUE THREAD

**Issue by PPzpower** [yuanqili78/SED-Net#7]
Hello, what does the shape of (N,2) for the predicted edge.txt mean? Or how can I visualize the predicted edges generated by the edge.txt?

**Comment by yuanqili78** [comment]
It is the lable of input points (is edge point or non-edge point)

**Comment by PPzpower** [comment]
> It is the lable of input points (is edge point or non-edge point)


Certainly! Thank you very much! I have a request. Do you have the code mentioned in the paper for reconstructing CAD models? If you do, could you please share it?
